index.bs

```diff
@@ -1138,12 +1140,15 @@ To <dfn>canonicalize a sanitizer element with attributes</dfn> a {{SanitizerElem
 
 1. Let |result| be the result of [=canonicalize a sanitizer element=] with |element|.
 1. If |element| is a [=dictionary=]:
-   1. [=list/iterate|For each=] |attribute| in
-      |element|["{{SanitizerElementNamespaceWithAttributes/attributes}}"]:
-      1. [=list/Append=] the result of [=canonicalize a sanitizer attribute=] with |attribute| to |result|["{{SanitizerElementNamespaceWithAttributes/attributes}}"].
-   1. [=list/iterate|For each=] |attribute| in
-      |element|["{{SanitizerElementNamespaceWithAttributes/removeAttributes}}"]:
-      1. [=list/Append=] the result of [=canonicalize a sanitizer attribute=] with |attribute| to |result|["{{SanitizerElementNamespaceWithAttributes/removeAttributes}}"].
+   1. If neither |element|["{{SanitizerConfig/attributes}}"] nor
+      |element|["{{SanitizerConfig/removeAttributes}}"] [=map/exist=], then [=map/set=]
+      |result|["{{SanitizerConfig/removeAttributes}}"] to &laquo; &raquo;.
+   1. If |element|["{{SanitizerElementNamespaceWithAttributes/attributes}}"] [=map/exists=]:
```

Comment:
This can be "Otherwise, if" then. Although maybe that is a little weird as we can't use it for the last one. Or maybe we can because we already ensured they are mutually exclusive at this point?

Reply:
Oh yeah, that'd be nice. https://infra.spec.whatwg.org/#iteration-continue